tests/build-and-run-tests.sh

```diff
@@ -28,9 +28,9 @@ function ciSkip {
     exit 0
 }
 
-# Finds the branching point of two commits. 
```

Comment:
It's very nice you're fixing these trailing spaces. Can you make this a separate commit though? As I mentioned before, we try to keep commits focused to what they describe they're doing and this change is unrelated to tests.